### PR TITLE
Rich console width

### PR DESCRIPTION
**Pull request by dbochkov-flexcompute**
Here's how we can adjust width of rich output, so that it should eliminate horizontal scroll bars in web documentation. Perhaps an alternative that we can consider is to define this width in the config, so that it could be changed as needed? What do you think? @tylerflex @momchil-flex

### CODE REVIEW

**Comment by dbochkov-flexcompute** [comment]
So, I realized we already create a console instance inside our logger, and decided just to retrieve it anytime we need a rich console. Do you see any issues with that?

**Comment by tylerflex** [comment]
No specifically. As long as it works properly when running different notebooks for example it should be fine.

File: tidy3d/log.py [hunk 1]
```diff
@@ -32,6 +32,8 @@
     "CRITICAL": "red bold",
 }
 
+CONSOLE_WIDTH = 80
```
Comment (tylerflex) [inline tidy3d/log.py:35]: can you just add a comment above this saying that
```py
Width of the console used for rich logging (in characters).
```

**Review by tylerflex — APPROVED** [review]
Looks good! Thanks! just a minor comment to explain the `CONSOLE_WIDTH` variable for future reference and also could you add a CHANGELOG item? Thanks!

**Comment by momchil-flex** [comment]
Can you just double check that the logging level and verbose in web functions are still independent from one another. I think they should be but just to make sure.

**Comment by tylerflex** [comment]
I'm excited to see the newly rendered docs after this fix :D

**Pull request merged**